src/plugins/ResizeReorder/ResizeReorderCell.js

```diff
@@ -0,0 +1,262 @@
+/**
+ * Copyright Schrodinger, LLC
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree. An additional grant
+ * of patent rights can be found in the PATENTS file in the same directory.
+ *
+ * @providesModule ResizeReorderCell
+ * @typechecks
+ * @file Wrapper component for ResizerKnob and ReorderHandle. It is responsible for translating cell in case of reordering
+ */
+
+import FixedDataTableCellDefault from '../../FixedDataTableCellDefault';
+import React from 'react';
+import PropTypes from 'prop-types';
+import ResizerKnob from './ResizerKnob';
+import ReorderHandle from './ReorderHandle';
+import joinClasses from '../../vendor_upstream/core/joinClasses';
+import cx from '../../vendor_upstream/stubs/cx';
+import { PluginContext } from '../../Context';
+
+const BORDER_WIDTH = 1;
+
+/**
+ * A plugin that can make use of ResizerKnob and ReorderHandle to provide the
+ * resize and reorder functionality to the columns.
+ *
+ * Pass this to the `header` prop of the `Column` to invoke resizer/reorder handle.
+ * `onColumnResizeEndCallback` and `onColumnReorderEndCallback` activate the respective
+ * functionality. Either one or both can be used together.
+ *
+ */
+class ResizeReorderCell extends React.PureComponent {
+
+  /**
+   * @param {JSX.Element} content
+   * @param {CSSProperties} style
+   * @param {string} className
+   * @returns {function(CSSProperties, React.ReactNode, boolean)}
+   */
+  renderReorderHandleContent = (content, style, className) =>
+    (reorderStyles, reorderHandle, isReordering) => {
+      const reorderClasses = joinClasses(className, cx({
+        'public/fixedDataTableCell/hasReorderHandle': true,
+        'public/fixedDataTableCell/reordering': isReordering,
+      }));
+      return <div className={reorderClasses} style={{ ...style, ...reorderStyles }}>
+        {reorderHandle}
+        {content}
+      </div>;
+    };
+
+  /**
+   * @param {JSX.Element} content
+   * @param {CSSProperties} style
+   * @param {string} className
+   * @returns {JSX.Element|null}
+   */
+  renderReorderHandle = (content, style, className) => {
+    if (!this.props.onColumnReorderEndCallback) {
+      return null;
+    }
+
+    return (
+      <ReorderHandle
+        render={this.renderReorderHandleContent(content, style, className)}
```

Comment:
Since we're now doing this for v2.0, we can just have ReorderHandle directly render the cell content, without making use of render props.

Reply:
Done